csharp/src/Google.Protobuf.Test/Google.Protobuf.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462;netcoreapp3.1;net60</TargetFrameworks>
+    <TargetFrameworks>net462;net6.0</TargetFrameworks>
```

Comment:
This change broke the GitHub action - please edit `.github/workflows/test_csharp.yml` as well to specify net6.0 instead of net60. (It's in two places.)

As far as I can see that should be the last net60 reference... we'll get there.